### PR TITLE
Replace headers with options in fetchGuillotine/fetchFromApi #486

### PR DESCRIPTION
**Pull request by pmi**
- also removed redundant eslintrc

### CODE REVIEW

File: src/guillotine/fetchContentData.ts [hunk 1]
```diff
@@ -20,7 +16,10 @@ export const fetchContentData = async <Content = Record<string, unknown>>(
     if (variables && Object.keys(variables).length > 0) {
         body.variables = variables;
     }
-    const contentResults = await fetchGuillotine<ContentResult<Content>>(contentApiUrl, body, projectConfig, headers);
+    const contentResults = await fetchGuillotine<ContentResult<Content>>(contentApiUrl, projectConfig, {
```
Comment (ComLock) [inline src/guillotine/fetchContentData.ts:19]: Why not go all the way and make a single "object" parameter?

File: src/guillotine/fetchFromApi.ts [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
-        },
-        body: JSON.stringify(body),
-    };
+    options?: FetchOptions,
+): Promise<GuillotineResponseJson> {
 
-    if (headers) {
-        Object.assign(options.headers, headers);
-    }
+    const opts = {
```
Comment (ComLock) [inline src/guillotine/fetchFromApi.ts:10]: Where did the type go? :RequestInit

**Review by ComLock — CHANGES_REQUESTED** [review]

File: src/guillotine/fetchContentData.ts (hunk 1 above)
Comment (pmi) [reply src/guillotine/fetchContentData.ts:19]: Just changed as little as possible, this was not really necessary here

File: src/guillotine/fetchFromApi.ts (hunk 2 above)
Comment (pmi) [reply src/guillotine/fetchFromApi.ts:10]: It is not assignable to RequestInit type because of the `next` field that is added by Next

**Pull request merged**